python/sdk/merlin/observability/inference.py

```diff
@@ -17,41 +17,59 @@ class ValueType(Enum):
 @dataclass
 class RegressionOutput:
     prediction_score_column: str
+    actual_score_column: str
 
     @property
     def column_types(self) -> Dict[str, ValueType]:
-        return {self.prediction_score_column: ValueType.FLOAT64}
+        return {
+            self.prediction_score_column: ValueType.FLOAT64,
+            self.actual_score_column: ValueType.FLOAT64
+        }
 
 
 @dataclass_json
 @dataclass
 class BinaryClassificationOutput:
     prediction_score_column: str
+    actual_score_column: str
     prediction_label_column: Optional[str] = None
+    actual_label_column: Optional[str] = None
 
     @property
     def column_types(self) -> Dict[str, ValueType]:
-        column_types_mapping = {self.prediction_score_column: ValueType.FLOAT64}
+        column_types_mapping = {
+            self.prediction_score_column: ValueType.FLOAT64,
+            self.actual_score_column: ValueType.FLOAT64
+        }
         if self.prediction_label_column is not None:
             column_types_mapping[self.prediction_label_column] = ValueType.STRING
+        if self.actual_label_column is not None:
+            column_types_mapping[self.actual_label_column] = ValueType.STRING
         return column_types_mapping
 
 
 @dataclass_json
 @dataclass
 class MulticlassClassificationOutput:
     prediction_score_columns: List[str]
+    actual_score_columns: List[str]
     prediction_label_columns: Optional[List[str]] = None
+    actual_label_column: Optional[str] = None
```

Comment:
For multi-class classification, there will be multiple score columns but only one label per row, correct? If so, `actual_label_column: Optional[str]` makes sense to me. But  should` prediction_label_columns: Optional[List[str]] = None` be a single string instead of List?